_templates/ripple/tide-module/workspace.json.t

```diff
@@ -0,0 +1,7 @@
+---
+inject: true
```

Comment:
This allows you to update an existing file and inject the content at a certain point (after is a regex)

Reply:
https://www.hygen.io/docs/templates#injection